plugins/remotefortressreader/CMakeLists.txt

```diff
@@ -41,4 +42,4 @@ ELSE(UNIX)
     )
 ENDIF(UNIX)
 # this makes sure all the stuff is put in proper places and linked to dfhack
-DFHACK_PLUGIN(RemoteFortressReader ${PROJECT_SRCS} LINK_LIBRARIES ${PROJECT_LIBS} PROTOBUFS ${PROJECT_PROTO} )
+DFHACK_PLUGIN(RemoteFortressReader ${PROJECT_SRCS} LINK_LIBRARIES protobuf-lite ${PROJECT_LIBS} COMPILE_FLAGS_MSVC "/FI\"Export.h\"" COMPILE_FLAGS_GCC "-include Export.h -Wno-misleading-indentation" )
```

Comment:
What's with the `-include` here? Shouldn't the plugin source file(s) include that?